index.html

```diff
@@ -34,6 +34,24 @@
     </div>
 </section>
 
+<section id="la-pull-request">
+    <h1>C'est quoi une PR ?</h1>
```

Comment:
I propose to merge this as it is and will enhance this later while the whole squeleton is on !